modules/tinymce/src/plugins/advlist/main/ts/api/Options.ts

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright (c) Tiny Technologies, Inc. All rights reserved.
+ * Licensed under the LGPL or a commercial license.
+ * For LGPL see License.txt in the project root for license information.
+ * For commercial licenses see https://www.tiny.cloud/
+ */
+
+import { Type } from '@ephox/katamari';
+
+import Editor from 'tinymce/core/api/Editor';
+import { EditorOptions } from 'tinymce/core/api/OptionTypes';
+
+const option: {
+  <K extends keyof EditorOptions>(name: K): (editor: Editor) => EditorOptions[K] | undefined;
+  <T>(name: string): (editor: Editor) => T | undefined;
+} = (name: string) => (editor: Editor) =>
+  editor.options.get(name);
+
+const register = (editor: Editor): void => {
+  const registerOption = editor.options.register;
+
+  const stringListProcessor = (value: unknown) => {
```

Comment:
This one is a candidate for a core processor it's fairly common that we have lists like this.

Reply:
Honestly there's a few I've noticed that could be converted to built-in processors, this is definitely one and so is `object[]`.

Reply:
Good to know we're all on the same page (even if that means we're making the same comment on multiple PRs).

Reply:
As discussed at stand up today we've made the `string[]` allow both a string array and a comma/space separated string.